osu.Game/Overlays/Dashboard/Friends/FriendDisplay.cs

```diff
@@ -175,127 +156,47 @@ private void load(OverlayColourProvider colourProvider, IAPIProvider api)
 
             background.Colour = colourProvider.Background4;
             controlBackground.Colour = colourProvider.Background5;
-
-            apiFriends.BindTo(api.Friends);
-            apiFriends.BindCollectionChanged((_, _) => Schedule(() => Users = apiFriends.Select(f => f.TargetUser).ToList()), true);
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            onlineStreamControl.Current.BindValueChanged(_ => recreatePanels());
-            userListToolbar.DisplayStyle.BindValueChanged(_ => recreatePanels());
-            userListToolbar.SortCriteria.BindValueChanged(_ => recreatePanels());
-            searchTextBox.Current.BindValueChanged(_ =>
-            {
-                if (currentContent.IsNotNull())
-                    currentContent.SearchTerm = searchTextBox.Current.Value;
-            });
-        }
-
-        private void recreatePanels()
-        {
-            if (!users.Any())
-                return;
-
-            cancellationToken?.Cancel();
-
-            if (itemsPlaceholder.Any())
-                loading.Show();
```

Comment:
In the new code, the `if` is not present - the loading spinner is always shown at this point.

This was a conscious change, because one of my tests was adding a `Thread.Sleep(2000)` in `FriendsList.load()` and seeing how it behaved, and it felt wrong to see a blank display with N>0 friends.

Reply:
Seems fine.